CHANGELOG.md

```diff
@@ -0,0 +1,125 @@
+# Changes
+
+- 0.27.0 - Improve MD011/MD013/MD022/MD031/MD032/MD033/MD034/MD040/MD043/MD051/
+           MD053, generate/separate documentation, improve documentation, update
+           dependencies.
+- 0.26.2 - Improve MD037/MD051/MD053.
```

Comment:
By flipping the order, patch updates are no longer associated with minor updates as they were before as nested list items.

Reply:
Addressed in other comment thread.